### ISSUE TITLE
Add read-only visualization of Components/Entities that are not instantiated

### ISSUE THREAD

**Issue by leonardo-ourmachinery** [OurMachinery/themachinery-public#447]
We should display a "read-only" view of the properties of the selected components/entities in the property tab.
The user should be able to see those values but not edit them without first overriding the Entity/Component.

**Comment by niklas-ourmachinery** [comment]
I agree that this would be nice, but this is potentially complicated to implement.

The properties view depends a lot on custom overrides `tm_properties_aspect_i->custom_ui()`.

For a "read-only" mode to work consistently across the board, all those custom implementations must also support a "read-only" mode. I.e., drawing the controls grayed out and not interactable.
